package.json

```diff
@@ -58,6 +58,7 @@
     "publish:next": "npm publish dist/clr-ui --tag next; npm publish dist/clr-angular --tag next; npm publish dist/clr-icons --tag next",
     "publish:local": "npm publish dist/clr-ui --registry http://localhost:4873; npm publish dist/clr-angular --registry http://localhost:4873; npm publish dist/clr-icons --registry http://localhost:4873",
     "ks:publish": "ng build --prod ks-app; cat ./dist/ks-app/index.html > ./dist/ks-app/404.html; surge -p ./dist/ks-app",
+    "build:dev": "ng build --prod",
```

Comment:
This line made me laugh.

Reply:
This is for the netlify script, wanting to keep it consistent.